data/json/npcs/EOC_talkers/portal_storm.json

```diff
@@ -109,6 +109,12 @@
     "type": "talk_topic",
     "id": "TALK_PORTAL_STORM_WILL_IT",
     "dynamic_line": "&You do not exist anymore.  Something else exists elsewhere, and vaguely remembers being you.  Its (torment?/joy?/confusion?/freedom?) will never truly end.",
-    "responses": [ { "text": "An ending?  The cataclysm was bad but was this better?", "topic": "TALK_DONE", "effect": [ "u_die" ] } ]
+    "responses": [
+      {
+        "text": "An ending?  The cataclysm was bad but was this better?",
+        "topic": "TALK_DONE",
+        "effect": [ "u_die", { "u_add_effect": "a_winner_is_u", "duration": "PERMANENT" } ]
```

Comment:
just curious, but applying effect works for dead character?

Reply:
Apparently yes, I didn't even consider that it could be an issue. It might be because we check for this effect before the character is erased.